tools/RAiDER/models/gmao.py

```diff
@@ -71,7 +71,7 @@ def _fetch(self, out):
         time1 = time
         time = round_time(time, 3 * 60 * 60)
         if not time1 == time:
-            logger.warning('Rounded given hour from  %d to %d. May be incorrect near beginning/end of day.', time1.hour, time.hour)
```

Comment:
Please use some more descriptive variable names like `original_time` and `rounded_time`?